chia/data_layer/data_layer.py

```diff
@@ -298,7 +306,7 @@ async def subscribe(self, store_id: bytes32, mode: DownloadMode, ip: str, port:
     async def unsubscribe(self, tree_id: bytes32) -> None:
         subscriptions = await self.get_subscriptions()
         if tree_id not in [subscription.tree_id for subscription in subscriptions]:
```

Comment:
By changing the bracket, this can return `True` as soon as the validating case is found.

```python
def f(n):
    print(n)
    return n

3 in [f(n) for n in range(10)]  # Prints all ten numbers
3 in (f(n) for n in range(10))  # Prints numbers up to 3
```